scilpy/tractograms/streamline_operations.py

```diff
@@ -656,6 +656,8 @@ def resample_streamlines_step_size(sft, step_size):
 
     # Return to original space
     resampled_sft.to_space(orig_space)
+    resampled_sft.streamlines._data = resampled_sft.streamlines._data.astype(
```

Comment:
I thought accessing `._data` was forbidden or highly discouraged.

Reply:
Yes, but nibabel default _data dtype is now using `float` so it default to float64 which crash our cython. I don't want to play with cython.

(It is discouraged to slice, overwrite, reshape or shuffle. But I feel that a cast isn't as bad)

Reply:
Ok. I think eventually you should add this to dipy's stateful tractogram. Like a sft.to_dtype method.